java-common-protos/.repo-metadata.json

```diff
@@ -6,12 +6,12 @@
   "client_documentation": "https://cloud.google.com/java/docs/reference/proto-google-common-protos/latest/history",
   "release_level": "stable",
   "transport": "grpc",
-  "requires_billing": true,
   "language": "java",
   "repo": "googleapis/sdk-platform-java",
   "repo_short": "java-common-protos",
-  "library_type": "OTHER",
   "distribution_name": "com.google.api.grpc:proto-google-common-protos",
+  "library_type": "OTHER",
+  "requires_billing": true,
   "excluded_dependencies": "proto-google-common-protos,grpc-google-common-protos,proto-google-common-protos-parent",
   "excluded_poms": "proto-google-common-protos-bom,proto-google-common-protos"
-}
+}
```

Comment:
Do we know what is the difference here?

Reply:
Ah, there's an additional blank line in the main branch. Since it was not generated before, I think this line is added by hand.